services/API-service/README.md

```diff
@@ -1,114 +1,16 @@
 # IBF-API-service: REST API of the IBF-system
 
-This service creates endpoints which serves data from the IBF-database to (for example) the IBF-interfaces.
+The [IBF API](https://ibf.510.global/docs) is the REST API of the IBF-system, which allows the IBF pipelines (a.o.) to write data to the database and the [IBF dashboard](https://ibf.510.global) (a.o.) to read data from the database.
```

Comment:
what is a.o. ?

Reply:
I think it is fine, only I didn't know what a.o. meant.

The pipelines also read from the database, and the dashboard also writes to the database.

The description is good as is because it sufficiently conveys the idea.